lib/views/activites/count_activity_view.dart

```diff
@@ -23,153 +24,238 @@ class CountActivityView extends StatefulWidget {
-                    child: Row(
-                      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                      children: <Widget>[
-                        Transform.scale(
-                          scale: 1.5,
-                          child: IconButton(
-                            icon: Icon(
-                              FontAwesomeIcons.chevronLeft,
-                              color: Colors.white,
-                            ),
-                            onPressed: () {
-                              if (count > 1) {
-                                setState(() {
-                                  count = count - 1;
-                                });
-                              }
-                            },
-                            color: Colors.white,
-                          ),
+                  ],
+                ),
+              ),
+              
+        !widget.isReview ? Expanded(child: Container()) : Container(),
+        widget.isReview
+            ? Container()
+            : Container(
+                width: Screen.width(context),
+                color: Theme.of(context).primaryColorDark,
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: <Widget>[
+                    Padding(
+                      padding: const EdgeInsets.all(8.0),
+                      child: OutlineButton(
+                        onPressed: () => Navigator.of(context).pop(),
+                        borderSide: BorderSide(
+                          color: HexColor("FF777777"),
                         ),
-                        Text(
-                          count.toString(),
-                          style: TextStyle(
-                            fontSize: Screen.width(context) <= 350 ? 40 : 60,
-                            color: Colors.white,
-                          ),
+                        shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.circular(5.0),
                         ),
-                        Transform.scale(
-                          scale: 1.5,
-                          child: IconButton(
-                            icon: Icon(
-                              FontAwesomeIcons.chevronRight,
-                              color: Colors.white,
-                            ),
-                            onPressed: () {
-                              if (count < 100) {
-                                setState(() {
-                                  count = count + 1;
-                                });
-                              }
-                            },
-                          ),
+                        child: Body(
+                          'Pass',
+                          size: Screen.width(context) >= 600
+                              ? 30
+                              : Screen.width(context) <= 350 ? 16 : 20,
```

Comment:
Is this somewhere you'd want to use your Screen class methods?

Reply:
Yes it is

Reply:
fixed